spec/models/game_spec.rb

```diff
@@ -74,4 +74,61 @@
       expect(game.active_player).to eq(game.black_player)
     end
   end
+
+  describe '#stalemate?' do
+    before do
+      allow_any_instance_of(Game).to receive(:populate_board!).and_return true
+    end
+
+    let(:game) { FactoryGirl.create(:game) }
+    let(:white_king) { FactoryGirl.create(:king, type: 'King', color: 'white', row: 7, column: 3) }
+    let(:black_king) { FactoryGirl.create(:king, type: 'King', color: 'black', row: 0, column: 3) }
+
+    it 'should return true if active player is in stalemate' do
+      game.pieces << white_king
+      game.pieces << black_king
+      game.pieces << FactoryGirl.create(:black_pawn, type: 'Pawn', row: 6, column: 3)
+      game.pieces << FactoryGirl.create(:white_pawn, type: 'Pawn', row: 1, column: 3)
```

Comment:
haha yeah no worries it's definitely tricky. You could potentially just look at the wikipedia page for chess stalemate and I think they have some example pictures of stalemate situations so you could then just copy that layout for you test if you want.